jsk_data/node_scripts/data_collection_server.py

```diff
@@ -86,7 +87,10 @@ def __del__(self):
             sub.unregister()
 
     def sub_cb(self, msg, topic_name):
-        self.msg[topic_name] = msg
+        self.msg[topic_name] = {
+                'stamp': msg.header.stamp if msg._has_header else rospy.Time.now(),
```

Comment:
no reason for 8 spaces, but no reason for 4 spaces because there is no lint error.
i read python style code, and it says continuous line indent is optional rule, which i didn't know.